components/suggestion/suggestion_list.jsx

```diff
@@ -209,12 +211,23 @@ export default class SuggestionList extends React.PureComponent {
 
         const mainClass = 'suggestion-list suggestion-list--' + this.props.location;
         const contentClass = 'suggestion-list__content suggestion-list__content--' + this.props.location;
+        let suggestionListMaxHeight = '';
+
+        suggestionListMaxHeight = windowHeight() - (this.props.wrapperRef.clientHeight + Constants.PREVIEWER_HEIGHT);
+        if (suggestionListMaxHeight > Constants.SUGGESTION_LIST_MAXHEIGHT) {
+            suggestionListMaxHeight = Constants.SUGGESTION_LIST_MAXHEIGHT;
+        }
+
+        const contentStyle = {
+            maxHeight: suggestionListMaxHeight,
```

Comment:
minor nitpick: if we rename `suggestionListMaxHeight` to `maxHeight` we can then do:

```js
const contentStyle = { maxHeight };
```